src/execution/sql/ddl_executors.cpp

```diff
@@ -152,20 +151,23 @@ bool DDLExecutors::CreateIndex(const common::ManagedPointer<catalog::CatalogAcce
   const auto &schema = accessor->GetIndexSchema(index_oid);
   // Instantiate an Index and update the pointer in the Catalog
   storage::index::IndexBuilder index_builder;
-  index_builder.SetSqlTableAndTransactionContext(accessor->GetTable(table), populate_txn);
   index_builder.SetKeySchema(schema);
   auto *const index = index_builder.Build();
-  index->SetNotLive();
 
-  bool result UNUSED_ATTRIBUTE = accessor->SetIndexPointer(index_oid, index);
-  TERRIER_ASSERT(result, "CreateIndex succeeded, SetIndexPointer must also succeed.");
+  if (concurrent) {
+
```

Comment:
I guess here is a TODO for the concurrent case?

Reply:
This is now an assert